planetary.lua

```diff
@@ -70,10 +70,10 @@ function init()
     sc.rate(i,1.0)
     sc.play(i,0)
 
-    sc.post_filter_dry(i,0)
+    sc.post_filter_dry(i,0.5)
```

Comment:
were these changes to @infovore 's script intentional?

Reply:
oh sorry, no they wern't